.github/actions/setup-kubernetes/action.yaml

```diff
@@ -110,10 +110,22 @@ runs:
       kind: InitConfiguration
       nodeRegistration:
         criSocket: "unix:///var/run/crio/crio.sock"
+        kubeletExtraArgs:
+          runtime-cgroups: /system.slice/crio.service
       ---
-      kind: KubeletConfiguration
       apiVersion: kubelet.config.k8s.io/v1beta1
-      cgroupDriver: cgroupfs
+      kind: KubeletConfiguration
+      cgroupDriver: systemd
+      cgroupRoot: /
+      systemCgroups: /system.slice
+      kubeletCgroups: /system.slice/kubelet.service
+      enforceNodeAllocatable:
+      - pods
+      - system-reserved
+      systemReservedCgroup: /system.slice
+      kubeReservedCgroup: /system.slice/kubelet.service
+      systemReserved:
+        ephemeral-storage: 1Gi
       EOF
```

Comment:
add `hairpinMode: "hairpin-veth"`, Scylla requires it for TRUNCATE queries

Reply:
thx, pls save it for the other "tune PR" as I was asked to keep this only for the slices fix

Reply:
included in https://github.com/scylladb/scylla-operator/pull/1169